content/articles/how-to-recieve-emails-using-emailjs-in-vue3/index.md

```diff
@@ -0,0 +1,382 @@
+# How to recieve emails using Emailjs in Vue3
+
+## **Introduction**
```

Comment:
Hi, just to ensure your article is well polished, remove the spacings between the heading and the first sentence. This should be done across the entire article.

. Ensure you use the correct headers i.e use level 3 (###) and not ##
. Remove the bold feature on the header i.e, instead of  '**Introduction**', use 'Introduction'.

> Ensure these edits are done on the entire article!